source/slang/type-layout.cpp

```diff
@@ -1460,9 +1487,11 @@ bool doesResourceRequireAdjustmentForArrayOfStructs(LayoutResourceKind kind)
 //
 // and then we expect `foo_b` to get `register(t8)`, rather
 // than `register(t1)`.
-static RefPtr<TypeLayout>maybeAdjustLayoutForArrayElementType(
+//
+static RefPtr<TypeLayout> maybeAdjustLayoutForArrayElementType(
     RefPtr<TypeLayout>  originalTypeLayout,
-    UInt                elementCount)
+    LayoutSize          elementCount,
+    UInt&               ioAdditionalSpacesNeeded)
```

Comment:
It was suggested for call site clarity that an out/inOut param might be better served as a pointer?